roles/hostapd/defaults/main.yml

```diff
@@ -45,4 +46,5 @@ hostapd__configs:
     hostapd__wpa: ''
     hostapd__wpa_key_mgmt: ''
     hostapd__wpa_pairwise: ''
-
+    #hostapd__vlan_file_content: ''
+    #hostapd__vlan_file: ''
```

Comment:
I'm not sure Ansible will work if we keep defaults like this even if there is a conditional in `Add vlan_file for hostapd` task.

Reply:
I had them here to show all options. I could provide them with values that will not change the default config.

The idea is that if hostapd__vlan_file is there is needs to be defined and not empty. So, if the variable hostapd__vlan_file is defined but empty, it should failed. I will had a test of not empty.

The hostapd__vlan_file_content of the hostapd__vlan_file could be empty. It has a default mapping.
```
# If no entries are provided by this file, the station is statically mapped
# to <bss-iface>.<vlan-id> interfaces.
# Each line can optionally also contain the name of a bridge to add the VLAN to
 ```
[source: https://w1.fi/cgit/hostap/plain/hostapd/hostapd.conf ]

We also could add a test around dynamic_vlan. The dynamic_vlan is defined and > 0, then hostapd__vlan_file should be there and not empty.

Reply:
Looks good to me. I had a minor comment.